contents/blog/stop-treating-engineers-differently.md

```diff
@@ -0,0 +1,68 @@
+---
+date: 2022-09-07
+title: "Startups, please stop treating engineers like a different kind of person"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Inside PostHog"]
+author: ["charles-cook"]
+featuredImage: ../images/blog/transparency.jpeg
+featuredImageType: full
+---
+
+Today I’d like to ~~rant~~ talk about how non-engineering people at startups – especially execs – seem to treat engineers like a fundamentally different type of human being. A whole new species, with a different set of personal motivations, goals, frustrations, and passions to the rest of the company.
+
+Why does this bizarre behavior occur? And why does it matter?
+
+I suspect that a lot of it is down to non-engineers not speaking the language of technical work, so therefore the people doing the work must be a different type of person.
+
+(Conversely, the fact that things like recruitment and brand marketing are easily understood by non-domain experts means that everyone has an opinion on them at a startup, but that’s a whole other conversation...)
+
+Non-engineers have built up years of pre-baked assumptions about how engineers work and what they are like that don’t really stand up to scrutiny. These are some of the most commonly-repeated myths I’ve heard. I’ve focused on startup culture because that’s what I know, but I imagine some of this stuff is relevant to larger companies too. 
```

Comment:
```suggestion
Non-engineers have built up years of pre-baked assumptions about how engineers work and what they are like that don’t stand up to scrutiny. Each of these myths have their unique downsides which only perpeptuate wrong-headed thinking and worse working environments. 

I’ve focused on startup culture because that’s what I know, but I imagine some of this stuff is relevant to larger companies too. 
```

Reply:
I'd prefer to leave my original words here if that's ok - your edits don't really sound like something I'd say and I think the big company thing is a distraction